docs/delegated-routing.md

```diff
@@ -1,11 +1,33 @@
-# New multi-router configuration system
+# Delegated Routing Notes
+
+- Status Date: 2025-12
+
+> [!IMPORTANT]
+> Most users are best served by setting delegated HTTP router URLs in [`Routing.DelegatedRouters`](https://github.com/ipfs/kubo/blob/master/docs/config.md#routingdelegatedrouters) and `Routing.Type` to `auto` or `autoclient`, rather than using custom routing with `Routing.Routers` and `Routing.Methods` directly.
+>
+> The rest of this documentation describes experimental features intended only for researchers and advanced users.
+
+----
+
+# Custom Multi-Router Configuration (Experimental)
 
 - Start Date: 2022-08-15
 - Related Issues:
   - https://github.com/ipfs/kubo/issues/9188
   - https://github.com/ipfs/kubo/issues/9079
   - https://github.com/ipfs/kubo/pull/9877
 
+> [!CAUTION]
+> **`Routing.Type=custom` with `Routing.Routers` and `Routing.Methods` is EXPERIMENTAL.**
+>
+> This feature is provided for **research and testing purposes only**. It is **not suitable for production use**.
+>
+> - The configuration format and behavior may change without notice between Kubo releases.
+> - Bugs and regressions affecting custom routing may not be prioritized or fixed promptly.
+> - HTTP-only routing configurations (without DHT) cannot reliably provide content to the network (👉️ see [Limitations](#limitations) below).
+>
+> **For production deployments**, use `Routing.Type=auto` (default) or `Routing.Type=autoclient` with [`Routing.DelegatedRouters`](https://github.com/ipfs/kubo/blob/master/docs/config.md#routingdelegatedrouters).
+
 ## Summary
```

Comment:
I think this Summary section is no longer helpful. The note above contains all the important information users need to know now.